### PR TITLE
Added AddButton, Header, LogoutButton, SearchFrom components. Changed…

### PR DESCRIPTION
**Pull request by ritasumtsova**
1. Added AddButton, Header, LogoutButton, SearchForm components with markup & a little very basic styles.
2. Added Sass (SCSS syntax) instead of CSS to the project.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* {
+  text-align: center;
+  padding: 5px;
```
Comment (didevgen) [inline src/App.scss]: not sure all elements need to have such rules :)

File: src/components/Department/Department.jsx [hunk 2]
```diff
@@ -25,7 +26,8 @@ export default class Department extends Component {
   }
 
   getDepartmentInfo = async () => {
-    const { match: { params: { id } } } = this.props || null;
+    // eslint-disable-next-line react/destructuring-assignment
```
Comment (didevgen) [inline src/components/Department/Department.jsx]: just remove the rule

**Review by didevgen — APPROVED** [review]

File: src/App.scss (hunk 1 above)
Comment (ritasumtsova) [reply src/App.scss]: agree :)

File: src/components/Department/Department.jsx (hunk 2 above)
Comment (ritasumtsova) [reply src/components/Department/Department.jsx]: +
I've fixed your comments and have done some additional changes

**Pull request merged**